jaeger_client/constants.py

```diff
@@ -33,10 +33,10 @@
 DEFAULT_FLUSH_INTERVAL = 1
 
 # Name of the HTTP header used to encode trace ID
-TRACE_ID_HEADER = b'uber-trace-id'
```

Comment:
And this was the error stack trace
```
File "opentracing_instrumentation/client_hooks/urllib2.py", line 97, in https_open
    return self.do_open(req, httplib.HTTPSConnection)
  File "opentracing_instrumentation/client_hooks/urllib2.py", line 54, in do_open
    resp = urllib2.AbstractHTTPHandler.do_open(self, conn, req)
  File "python2.7/urllib2.py", line 1174, in do_open
    h.request(req.get_method(), req.get_selector(), req.data, headers)
  File "python2.7/httplib.py", line 966, in request
    self._send_request(method, url, body, headers)
  File "python2.7/httplib.py", line 1000, in _send_request
    self.endheaders(body)
  File "python2.7/httplib.py", line 962, in endheaders
    self._send_output(message_body)
  File "python2.7/httplib.py", line 820, in _send_output
    msg += message_body
```

The `+=` is what was failing when the headers were defined as unicode strings.